src/test/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PerformerNameAndAddressTest.java

```diff
@@ -0,0 +1,96 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import org.junit.jupiter.api.Test;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
This is a new class so we can afford to be careful not to mix and match assertion libraries. Let's keep it consistent

Reply:
The only issue I can see with this is the `usingRecursiveComparison` tests where we're testing whether an object is equal to another object that is generated using `toString` i.e

```
        assertThat(performerNameAndAddress)
            .usingRecursiveComparison()
            .isEqualTo(RequesterNameAndAddress.fromString("NAD+SLA+A2442389:902++DR J SMITH"));
```

Reply:
From discussion in Teams, we understand the concern is that jupiter doesn't have an equivalent for `usingRecursiveComparison()` - and the agreed action then is to replace jupiter assertions with assertj, where possible. Is that right, @saileshp-kainos ?

Reply:
Done in 4505230